src/app/catalogue-search/catalogue-search-form/catalogue-search-form.component.ts

```diff
@@ -34,7 +34,7 @@ import { MatFormFieldAppearance } from '@angular/material/form-field';
 export class CatalogueSearchFormComponent implements OnInit {
   @Input() appearance: MatFormFieldAppearance = 'outline';
   @Input() routeSearchTerm?: string = '';
-  formGroup: FormGroup = new FormGroup({});
+  formGroup: UntypedFormGroup = new UntypedFormGroup({});
```

Comment:
I've heard about these new typed Angular Forms, would be interesting to refactor some of these at a later date.

Reply:
Yes - the automatic migration didn't work for me so I had to figure this out myself, and decided Untyped versions would be best for now.  But I've added a ticket #742 for sorting it out later